planetary-social/nos
# Add "Display" to the Name textfield header on the edit profile page.

## Issues covered
#1445 

## Description
This PR updates "Name" to "Display Name" in the `EditProfile` page
- The value of the `displayName` key was capitalised in the Localizable file
- The localized `displayName` key is used to replace the previous localized `name` key

## How to test
1. Build the code
2. Tap the Profile tab on the bottom right
3. Tap the Edit Profile button on the page
4. Please check that under the Basic Information category, the "Name" header has been changed to "Display Name".

## Screenshots/Video

| Before | After |
|--------|-------|
| <img src="https://github.com/user-attachments/assets/841e604d-7519-447a-9b0c-a58a1884f66e" alt="Before" width="250"/> | <img src="https://github.com/user-attachments/assets/b3c76c8e-3710-4214-8a59-e7e57e8940e8" alt="After" width="250"/> |
